impl/src/test/groovy/com/okta/sdk/impl/ds/WriteCacheFilterTest.groovy

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright 2017 Okta
```

Comment:
Can't you automate this somehow? ;)

Reply:
Yup, `mvn license:format` it only triggers on new files though, so I'll update the template, and the files in this PR directly